src/NServiceBus.RabbitMQ/AssemblyInfo.cs

```diff
@@ -1,9 +1,3 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
+﻿using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("NServiceBus.RabbitMQ")]
-[assembly: AssemblyCopyright("Copyright NServiceBus. All rights reserved")]
-[assembly: AssemblyProduct("NServiceBus.RabbitMQ")]
```

Comment:
https://docs.microsoft.com/en-us/dotnet/articles/core/tools/project-json-to-csproj talks about `<AssemblyTitle>My library</AssemblyTitle>` - is that the property which doesn't work properly?